wordpress-mobile/gutenberg-mobile
# Basic template picker

See https://github.com/WordPress/gutenberg/pull/18052 for description.

Fixes #1462 #1464 and partially addresses #1463

To test:

- iOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/12923
- Android: https://github.com/wordpress-mobile/WordPress-Android/pull/10775

Launch the apps and ensure the template picker is shown only on empty pages and when metro is running (`__DEV__ == true`)

Update release notes:
